main.py

```diff
@@ -1,280 +1,299 @@
-import random
-import os
+"""
+Caleb's Ciphernet
+End-to-end message encryption.
+"""
 import json
+import os
+import random
 
 
-sysrand = random.SystemRandom()
-current_user = ""
-user_data = {}
-home = 0
-Pass = 0
-New_account = 0
-Check_yes = ["yes", "y"]
-characters = ("a","b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "`", "~", "!", "@", "#", "$", "%", "^", "&", "*", "(", ")", "_", "-", "+", "=", "|", "[", "]", "{", "}", ";", ":", "'", ",", ".", "<", ">", "/", "?", " ", '"', "\\")
-alphabet = "abcdefghijklmnopqrstuvwxyz1234567890 -_=+[{]}|;:'\"/?.>,<!@#$%^&*()`~\\"
-
-while Pass == 0:
-  print("To log onto Ciphernet, please enter 1 to login.")
-  print("If you do not have a account, please enter 2 to create an account.")
-  Start = input().lower()
+ALPHABET = "abcdefghijklmnopqrstuvwxyz1234567890 -_=+[{]}|;:'\"/?.>,<!@#$%^&*()`~\\"
+SYS_RAND = random.SystemRandom()
+USER_DATA = {}
+HOME = 0
+PASS_ = 0
+NEW_ACCOUNT = 0
+CHECK_YES = ["yes", "y"]
 
-  Login = ["1", 1, "login", "log in"]
-  Signin = ["2", 2, "signin", "sign in", "create an account"]
+# sign in/up
+while PASS_ == 0:
+    print("To log onto Ciphernet, please enter 1 to login.")
+    print("If you do not have a account, please enter 2 to create an account.")
+    start = input().lower()
 
-  if Start in Login:
-    Pass = 1
-  elif Start in Signin:
-    Pass = 1
+    login = ["1", 1, "login", "log in"]
```

Comment:
Variable names should be lowercase. Did this in lots of places. Just commenting here.